packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md

```diff
@@ -5,42 +5,61 @@ title: Upgrade your Okta Sign-In Widget
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" />
 
-## Overview
+
+This guide covers how to upgrade the Okta Sign-In Widget, which depends on whether your user authentication deployment model is a redirect authentication or an embedded authentication.
+
+---
+**Learning outcomes**
+
+* Understand the benefits of keeping your Sign-In Widget up to date.
+* Update your Sign-In Widget based on your user deployment model.
+* Know how to obtain the latest version of the Sign-In Widget.
```

Comment:
It seems to me that logically, this should be the second bullet in the list — you understand why, then you learn how to obtain it, then you go and obtain it and update it.

Reply:
Fixed